### ISSUE TITLE
Instance: The "x replies" controls have visible text that is not included in their accessible name | WCAG 2.2 Violation - There are controls present within the interface with visible text that is not included as part of the controls' accessible names.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#36]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

#### Further information:

1. 

### Screenshot

![Screenshot of an example of one of the \`.chat-thread-toggle\` controls in its collapsed and expanded states. In both states, the control's visible text reads '1 reply'. In its collapsed state, the control's computed accessible name is 'Expand replies'. In its expanded state, the control's computed accessible name is 'Collapse replies'.](https://github.com/user-attachments/assets/06cbef6f-11d5-40d1-bb61-69c2011d2d9b)

### Code Reference

```
<!-- An example of one the controls in a collapsed state -->
<button class="chat-thread-toggle" […] aria-expanded="false" aria-label="Expand replies">
	[…]
	<span class="chat-reply-count">1 reply</span>
	[…]
</button>

<!-- The same control in an expanded state -->
<button class="chat-thread-toggle" […] aria-expanded="true" aria-label="Collapse replies">
	[…]
	<span class="chat-reply-count">1 reply</span>
	[…]
</button>
```
